### ISSUE TITLE
A11y_AzureDataStudioSQL_Projects_CreateNewProject_Keyboard: Keyboard focus is not visible on "Ok" button.

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/azuredatastudio#14336]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AzureDataStudioSQL_Feb2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#Keyboard;#WCAG2.4.7;#AINotInScope;
### Environment Details:
Application Name: Azure Data Studio
Version: 1.27.0-insider
Windows Version: Windows10

### Repro Steps:

1. Launch the application-Azure Database Project and install extension SQL Database Projects.
2. Tab till SQL Database Project Home Screen with heading "Projects".
3. Tab till "Create New Project" button or "+" button and hit enter.
4. Fill all the field and tab till "Ok" button and check if keyboard focus is visible on the ok button

### Actual:
Keyboard focus is not visible on "Ok" button.

Similar issue is present throughout the application where "Ok" button is present.

### Expected:
Keyboard focus should be visible on "Ok" button.

### User Impact:
Low vision users will not know where the keyboard focus goes on ok button or not.

**Comment by kisantia** [comment]
When using the High Contrast theme, it is clear when focus is on the ok button.

Focus on ok button: 
![image](https://user-images.githubusercontent.com/31145923/108794227-3c96c980-753a-11eb-8add-fb25be88d8c5.png)

focus not on ok button: 
![image](https://user-images.githubusercontent.com/31145923/108794235-41f41400-753a-11eb-85f1-95eba07bc043.png)

**Comment by Shivamd1** [comment]
Issue does not reproduces in latest 1.25.3 hence closing the bug

**Comment by prabhudevu** [comment]
**GithubTags**:#Closed;

**Comment by prabhudevu** [comment]
![14336_Fixed](https://user-images.githubusercontent.com/98821792/216963424-436338b1-c91b-4048-ba61-d7bbeb1365b1.png)

**GithubTags**:#Closed;
